### DISCUSSION TITLE
Azure Storage not utilized

### DISCUSSION THREAD

**Discussion by hucen0077** [directus/directus#5766] [Archive]
I have forked latest version of directus and have configured .env file to connect to azure sql server. With encryption it is working as expected.

However, when trying to connect to azure blob storage, it keeps ignoring the storage configurations set in .env file and uploads to local storage.

Here is the storage configuration that i have done.

STORAGE_LOCATIONS: 'azure'

STORAGE_AZURE_DRIVER: 'azure'
STORAGE_AZURE_CONTAINER_NAME: 'directus-dev'
STORAGE_AZURE_ACCOUNT_NAME: 'directustest'
STORAGE_AZURE_ACCOUNT_KEY: 'supersecretkey
STORAGE_AZURE_ENDPOINT: 'https://directustest.blob.core.windows.net/directus-dev'

**Comment by rijkvanzanten** [comment]
> it keeps ignoring the storage configurations set in .env file

This most definitely indicates that that configuration doesn't actually make it to Directus itself.

You're saying you're using a `.env` file, but that syntax wouldn't match what you'd expect in `.env`:

```
// .env
STORAGE_LOCATIONS="azure"

// ???
STORAGE_LOCATIONS: 'azure'
```

Where are you configuring these?
